src/pages/reports/ReportsPage.tsx

```diff
@@ -50,10 +46,38 @@ function ReportsPage() {
             accessor: column.accessor,
             Cell: ({ value }: any) => (value !== 'Many' ? formatDate(new Date(value), 'dd/MM/yyyy') : 'Many'),
           };
-        } if (currentColumnsLength < 4) {
+        } if (column.accessor === 'ShipmentVolume'
```

Comment:
Do we need such huge 'if' statement? Is there any way to 'shorcut' this 'if' statement?